test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -38,11 +38,18 @@ class test_uct_sockaddr : public uct_test {
         ASSERT_TRUE(sock_addr.addr != NULL);
 
         addr_in = (struct sockaddr_in *) (sock_addr.addr);
-        addr_in->sin_port = 0;   /* Use a random port */
+
+        /* Get a usable port on the host.
+         * Ports below 1024 are considered "privileged" (can be used only by user root).
+         * Ports above and including 1024 can use by anyone. */
+        do {
+            addr_in->sin_port = ucs::get_port();
+        } while (ntohs(addr_in->sin_port) < 1024);
```

Comment:
pls encapsulate this loop inside `ucs::get_port()` and do binding on random port > 1024